### ISSUE TITLE
Return how often the password was found

### ISSUE THREAD

**Issue by MichaelRumpler** [neoKushan/pwnedpasswords.net#1]
The web service also returns the number how often the given password was found in the database. But in your code you ignore that. It would be cool if you could return that number instead of just a bool to indicate how bad the password is.

**Comment by neoKushan** [comment]
Yup, I completely agree it would be a worthy addition. I've been trying to convince Troy Hunt to change the format of the response to a more standard one but to no avail, so I'll pencil this in for a future update.
